docs/conf.py

```diff
@@ -22,21 +22,27 @@
 import sphinx_rtd_theme
 from sphinx.ext.autodoc import between
 
+
+# Add package paths to the python path to allow sphinx to import modules
 sys.path.insert(0, os.path.abspath("../src/"))
 
+import whylogs
```

Comment:
By importing WhyLogs here you require the doc build runtime to have all whylogs dependencies. This is breaking `readthedocs` build in Sphinx (I didn't wire it in correctly so it's not showing up in the CI but I'm working on fixing it)

A better way to do this is here: https://github.com/whylabs/whylogs-python/blob/mainline/setup.py#L103

The runtime for docs building is here: https://github.com/whylabs/whylogs-python/blob/mainline/docs/requirements.txt

Reply:
Weird, it should find whylogs because of the line above where the whylogs path is added to the sys.path.  But I can wire something like that in.